wildlifelicensing/apps/applications/views/entry.py

```diff
@@ -312,7 +359,7 @@ def form_valid(self, form):
         application.applicant.save()
 
         # update any other applications for this user that are awaiting ID upload
-#       for application in Application.objects.filter(applicant_profile__user=applicant):
+        #       for application in Application.objects.filter(applicant_profile__user=applicant):
```

Comment:
The line was already there. It's just my auto-format that changed it.
Will remove.